internal/sweep/sweep_test.go

```diff
@@ -25,6 +27,7 @@ func TestMain(m *testing.M) {
 func addTestSweepers() {
 	connection.AddTestSweeper()
 	device.AddTestSweeper()
+	fabric_connection.AddTestSweeper()
```

Comment:
Not relevant for this PR or even for the Fabric team but the existing Metal sweepers in this file should all be renamed to `metal_<resource>`.